config/label-management.md

```diff
@@ -0,0 +1,126 @@
+# Managing cross-organization labels in existing repositories #
+
+This is a step-by-step guide/script for managing organization-wide github
+issue labels across all repositories.
+
+## Prerequisites ##
+
+- User has access to the github organization's repositories
+- User can generate a GitHub [Personal access token](https://github.com/settings/tokens)
+
+## Initial Setup ##
+
+1. Clone the repository locally
+
+```console
+git clone https://github.com/alta3/github-label-management.git
+cd github-label-management
+```
+
+1. Activate the virtual environment and install the requirements
```

Comment:
We usually prefer `pyenv` to `venv` these days.  I'm fine with leaving it how it is, but I just mention it in case you want to change it.

Reply:
I ~stole~ borrowed it from the cloned repo's README, so I'm happy to convert to `pyenv`.

Reply:
Resolved in https://github.com/cisagov/development-guide/pull/41/commits/105129a08608094e93141604071ab63eabb2dae1 - now I've ~stolen~ borrowed from our skeleton `setup_env` as well. :D

Reply:
[These lines](https://github.com/cisagov/development-guide/blob/label-mgmt/config/label-management.md#re-initializing-the-environment) also need some `pyenv`ification.

Reply:
Thank you for your attention to detail! Resolved in https://github.com/cisagov/development-guide/pull/41/commits/59da33d22408331ad21b329707ed8e83cb254391

Reply:
Thanks for adding commit 59da33d!